pom.xml

```diff
@@ -287,6 +287,13 @@
             <version>15.0</version>
         </dependency>
```

Comment:
Do we want to release this as 3.0.1 now, and have proctor point at 3.0.1? (and would that require all dependent projects using this to be bumped to point at 3.0.1 also, and would that bump be done in a different commit/review?)